.circleci/config.yml

```diff
@@ -360,13 +360,15 @@ commands:
           command: |
             psql -c "create user dockstore with password 'dockstore' createdb;" -U postgres
             psql -c 'create database webservice_test with owner = dockstore;' -U postgres
+            psql -c 'create database webservice_test_proposed with owner = dockstore;' -U postgres
```

Comment:
What's the use of the new database?

Reply:
I was getting an error in propose_migrations because dockstore wasn't the owner, and first tried fixing it here. But I should be able to revert it, and I will.